hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/streamer/HoodieFlinkStreamer.java

```diff
@@ -98,7 +98,7 @@ public static void main(String[] args) throws Exception {
     }
 
     DataStream<HoodieRecord> hoodieRecordDataStream = Pipelines.bootstrap(conf, rowType, parallelism, dataStream);
-    DataStream<Object> pipeline = Pipelines.hoodieStreamWrite(conf, parallelism, hoodieRecordDataStream);
+    DataStream<Object> pipeline = Pipelines.hoodieStreamWrite(conf, parallelism, hoodieRecordDataStream, conf.getString(FlinkOptions.TABLE_NAME));
     if (StreamerUtil.needsAsyncCompaction(conf)) {
```

Comment:
I mean you already passed in the `conf`, you can just fetch the table name inside `hoodieStreamWrite` and there is no need to pass around `conf.getString(FlinkOptions.TABLE_NAME)` again.

Reply:
yes ,  only the table name is passed here, 
but the table and column are passed in the HoodieTableSink.